### PR TITLE
V1.3

### PR DESCRIPTION
**Pull request by 7oSkaaa**
Make some changes in the script:

- Ask the user first if he wants to shorten the link or not.
- use Cuttly to shorten the link.
- change the ways to get contest_id and group_id in the script.
- make format to the files.
- sort imports.
- add .vscode folder to gitignore.

### CODE REVIEW

File: data/trainees.csv [hunk 1]
```diff
@@ -1,9 +1,52 @@
 names,handles
```
Comment (kerolloz) [inline data/trainees.csv]: Why are you including `trainees.csv` & `groups.txt`?

**Review by kerolloz — COMMENTED** [review]
I can also see permission changes. Any specific reason?

File: data/trainees.csv (hunk 1 above)
Comment (7oSkaaa) [reply data/trainees.csv]: It was included, for example, I updated it now with the handles in the last time.

Comment (kerolloz) [reply data/trainees.csv]: Yeah, I mean it should be ignored in the first place.

**Review by kerolloz — APPROVED** [review]

**Pull request merged**